Dockerfile

```diff
@@ -232,12 +233,16 @@ RUN \
   libmaxminddb-dev \
   certbot \
   docker-cli \
-  docker-compose \
   libgomp \
   && docker-php-ext-install sockets opcache pdo_mysql \
   && apk del .deps \
   && rm -rf /var/cache/apk/*
 
+RUN \
+  mkdir -p $DOCKER_CONFIG/cli-plugins \
+  && curl -SL https://github.com/docker/compose/releases/download/v2.5.0/docker-compose-linux-x86_64 -o $DOCKER_CONFIG/cli-plugins/docker-compose \
```

Comment:
Can we make the version number here as a variable on top?

Reply:
Does this also need to consider architecture as well instead of hardcoding `x86_64`?

Reply:
- Will make version ENV variable
- Don't think we need to consider architecture, right? It is all running in the same environment, inside a docker container. I will add it tho. Won't hurt, may help.

Reply:
I was not able to use the command suggested by @stnguyen90 , as it is a `bash` script, but Apwrite runs on `sh` (`/bin/sh: /bin/bash: not found`)

I replaced it with just `$(uname -m)` instead, WDYT?

Reply:
Nevermind, I think I figured it out. I used:

```sh
ARCH=$(uname -m) && if [ $ARCH == "armv7l" ]; then $ARCH="armv7"; fi
```

Reply:
> Don't think we need to consider architecture, right? It is all running in the same environment, inside a docker container. I will add it tho. Won't hurt, may help.

I think there are different images for different architectures and the matching image is used automatically. On my rpi:

```shell
$ uname -m
aarch64

$ docker run --rm -it appwrite/appwrite uname -m
aarch64
```

On my (intel) macbook:

```shell
$ uname -m                                      
x86_64

$ docker run --rm -it appwrite/appwrite uname -m
x86_64
```